support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/config/CassandraProperties.java

```diff
@@ -0,0 +1,44 @@
+package org.apereo.cas.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * This is {@link CassandraProperties}.
+ *
+ * @author David Rodriguez
+ * @since 5.0.0
+ */
+@ConfigurationProperties("cassandra")
```

Comment:
Fix the docs as well. I think you incorrectly somehow updated rest docs for this?